met-web/src/models/feedback.ts

```diff
@@ -0,0 +1,13 @@
+export interface Feedback {
+    rating: number;
+    comment: string;
+    commentType: string;
```

Comment:
We can specify this to the exact commentTypes commentType: 'Issue' | 'Idea' | 'Else'

Reply:
Implemented